app/build.gradle

```diff
@@ -0,0 +1,28 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 22
+    buildToolsVersion "22.0.1"
+
+    defaultConfig {
+        applicationId "com.google.devrel.samples.ttt"
+        minSdkVersion 14
```

Comment:
These days you should be build 17 as min. (Lowest Android Team recommends supporting)